packages-exp/auth-exp/src/core/strategies/action_code_settings.ts

```diff
@@ -18,20 +18,41 @@
 import { ActionCodeSettings } from '@firebase/auth-types-exp';
 
 import { GetOobCodeRequest } from '../../api/authentication/email_and_password';
+import { AuthErrorCode } from '../errors';
+import { assert } from '../util/assert';
+import { AuthCore } from '../../model/auth'; 
 
-export function setActionCodeSettingsOnRequest(
+export function setActionCodeSettingsOnRequest_(
```

Comment:
Internal methods should be prefixed with `_`, not suffixed

Reply:
mybad, I've been in the closure SDK too long